crucible-mir/src/Mir/PP.hs

```diff
@@ -310,6 +310,19 @@ instance Pretty BinOp where
       Unchecked op' -> pretty op' <> pretty "!"
       WithOverflow op' -> pretty op' <> pretty "?"
 
+instance Pretty VtableItem where
+  pretty (VtableItem fn def) =
+    vcat
+      [ pretty "(" <+> pretty fn
+      , pretty "," <+> pretty def <+> pretty ");"
+      ]
```

Comment:
I would flip these around, and maybe change the syntax to make it clearer that this is a key-value mapping.  `def` is the key and `fn` is the value, at least based on my interpretation of the `VtableItem` type:
```Haskell
data VtableItem = VtableItem
    { _vtFn :: DefId        -- ^ ID of the implementation that should be stored in the vtable
    , _vtDef :: DefId       -- ^ ID of the item definition in the trait
    }
```

Reply:
I've flipped these around and replaced the comma with `=>` to make it clearer that this is a key-value mapping.